timesketch/frontend-ng/README.md

```diff
@@ -0,0 +1,34 @@
+# Timesketch web frontend
```

Comment:
can we move that file to the `/docs/` folder instead?

Reply:
Not at this time - this is meant as a readme for anyone (dev) who are browsing the repo. I'll write more appropriate docs at a later stage

Reply:
ACK